server/auth_function/lambda_function.py

```diff
@@ -90,6 +89,10 @@ def lambda_handler(event: event_type.Event, context: Any) -> event_type.Event:
             audit_event_log["requestingUser"]["idpUserName"] = event["request"][
                 "userAttributes"
             ]["custom:idp_username"]
+        else:
+            audit_event_log["requestingUser"]["idpDisplayName"] = event["request"][
+                "userAttributes"
+            ]["custom:idp_display_name"]
```

Comment:
I see, I didn't notice this idp_user_id is in log already. Thanks for point that out.
Sure, then that seems to make sense with display_name; I assume it might be empty but does not matter.

Reply:
I just saw your comment haha, our test bcsc account has display name that's why I feel it might be good to show it, it's a readable name.